src/test/kotlin/se/uulm/snowballr/backend/repository/UserTableRepoTest.kt

```diff
@@ -214,4 +194,110 @@ class UserTableRepoTest : H2DatabaseTest(arrayOf(UserTable)) {
             assertThat(user1.id).isNotEqualTo(user2.id)
         }
     }
+
+    @Nested
+    inner class UpdateUser {
+        @ParameterizedTest(name = "Update the fields {0}")
+        @MethodSource("se.uulm.snowballr.backend.repository.UserTableRepoTest#validFieldMasks")
+        fun `When a user is updated, then only the fields specified in the field mask are updated and the updated user is returned`(
+            fieldMask: List<String>,
+        ) = testCoroutine {
+            val userId = insertTestUserAndGetId(email = "test.user@example.com")
+            val originalUser = repo.getUserById(userId)
+
+            val updatedUserBuilder = originalUser.toGrpcUser().toBuilder()
+
+            // Apply some updates to fields
+            if ("user.email" in fieldMask) updatedUserBuilder.email = "updated.user@example.com"
+            if ("user.first_name" in fieldMask) updatedUserBuilder.firstName = "John"
+            if ("user.last_name" in fieldMask) updatedUserBuilder.lastName = "Doe"
+            if ("user.role" in fieldMask) updatedUserBuilder.role = UserRole.USER_ROLE_ADMIN
+
+            val request = UserOuterClass.User.Update.newBuilder()
+                .setUser(updatedUserBuilder.build())
+                .setMask(FieldMaskUtil.fromStringList(fieldMask))
+                .build()
+
+            val updatedUser = repo.updateUser(request)
+
+            if ("user.email" in fieldMask) {
+                assertThat(
+                    updatedUser.email,
+                ).isEqualTo(
+                    "updated.user@example.com",
+                )
+            } else {
+                assertThat(updatedUser.email).isEqualTo("test.user@example.com")
+            }
+            if ("user.first_name" in fieldMask) {
+                assertThat(
+                    updatedUser.firstName,
+                ).isEqualTo("John")
+            } else {
+                assertThat(updatedUser.firstName).isEqualTo("Test")
+            }
+            if ("user.last_name" in fieldMask) {
+                assertThat(
+                    updatedUser.lastName,
+                ).isEqualTo("Doe")
+            } else {
+                assertThat(updatedUser.lastName).isEqualTo("User")
+            }
+            if ("user.role" in fieldMask) {
+                assertThat(
+                    updatedUser.role,
+                ).isEqualTo(
+                    UserRole.USER_ROLE_ADMIN,
+                )
+            } else {
+                assertThat(updatedUser.role).isEqualTo(UserRole.USER_ROLE_DEFAULT)
+            }
```

Comment:
These seem unnecessarily wrapped.

```suggestion
            if ("user.email" in fieldMask) {
                assertThat(updatedUser.email).isEqualTo("updated.user@example.com")
            } else {
                assertThat(updatedUser.email).isEqualTo("test.user@example.com")
            }
            if ("user.first_name" in fieldMask) {
                assertThat(updatedUser.firstName).isEqualTo("John")
            } else {
                assertThat(updatedUser.firstName).isEqualTo("Test")
            }
            if ("user.last_name" in fieldMask) {
                assertThat(updatedUser.lastName).isEqualTo("Doe")
            } else {
                assertThat(updatedUser.lastName).isEqualTo("User")
            }
            if ("user.role" in fieldMask) {
                assertThat(updatedUser.role).isEqualTo(UserRole.USER_ROLE_ADMIN)
            } else {
                assertThat(updatedUser.role).isEqualTo(UserRole.USER_ROLE_DEFAULT)
            }
```

Reply:
You are right, this was automatically formatted by the formatter, I do not quite understand why.